source/features/toggle-everything-with-alt.tsx

```diff
@@ -43,15 +43,27 @@ function resolvedCommentsSelector(clickedItem: HTMLElement): string {
 	return `.js-resolvable-thread-toggler[aria-expanded="${clickedItem.getAttribute('aria-expanded')!}"]:not(.d-none)`;
 }
 
+function expandSelector(): string {
+	return '.js-expand-full';
+}
+
+function collapseSelector(): string {
+	return '.js-collapse-diff';
+}
+
 function init(): void {
 	// Collapsed comments in PR conversations and files
 	delegate(document, '.minimized-comment details summary', 'click', clickAll(minimizedCommentsSelector));
 
 	// "Load diff" buttons in PR files
-	delegate(document, '.js-file .js-diff-load', 'click', clickAll(allDiffsSelector));
+	delegate(document, '.js-file .js-diff-load', 'click', clickAll(diffsSelector));
 
 	// Review comments in PR
 	delegate(document, '.js-file .js-resolvable-thread-toggler', 'click', clickAll(resolvedCommentsSelector));
+
+	// "Expand all" and "Collapse expanded lines" buttons in commit files
+	delegate(document, '.js-expand-full', 'click', clickAll(expandSelector));
+	delegate(document, '.js-collapse-diff', 'click', clickAll(collapseSelector));
```

Comment:
Let's make these a little more specific. Untested example:

```suggestion
	delegate(document, '.js-file .js-expand-full', 'click', clickAll(expandSelector));
	delegate(document, '.js-file .js-collapse-diff', 'click', clickAll(collapseSelector));
```